helm-system-packages-dpkg.el

```diff
@@ -245,6 +245,44 @@ Otherwise display in `helm-system-packages-buffer'."
         (push (match-string 1 url) urls)))
     (helm-system-packages-browse-url urls)))
 
+(defvar helm-system-packages-dpkg--source-name "dpkg source")
+
+(defvar helm-system-packages-dpkg--descriptions-global nil
+  "All descriptions.
+Used to restore complete description list when browsing dependencies.")
+(defvar helm-system-packages-dpkg--names-global nil
+  "All names.
+Used to restore complete name list when browsing dependencies.")
+
+(defun helm-system-packages-dpkg-deps (_candidate &optional reverse)
+  "Run a Helm session over the packages returned by COMMAND run over `helm-marked-candidates'.
+With prefix argument, insert the output at point.
+
+If REVERSE is non-nil, show reverse dependencies instead."
+  (setq helm-system-packages-dpkg--descriptions (or helm-system-packages-dpkg--descriptions-global helm-system-packages-dpkg--descriptions)
```

Comment:
Please fix all your +80 lines in your source files, it becomes unreadable.  Thanks.